README.md

```diff
@@ -81,6 +81,9 @@ Call git log on current highlighted branch with `[space]`
 |`space`| Git log |
 |`q, C-c, esc`| Quit |
 
+## Settings
+You can change app settings by editing `~/.config/configstore/check-it-out.json`
```

Comment:
Let's change this verbiage to something like 

> Configurable settings can be found at   `~/.config/configstore/check-it-out.json`
> Options include:
> * themeColor
>    Type: String
>    A hex color code to style Check It Out
> * logAlias...

Reply:
We should provide a base template for the user here as well. Probably will be identical to the defaults.

Reply:
Hey there, since we have only gitLogArguments, I think it will be better to provide with an example line. When we have a few additions, we can add a template. What do you think?

Reply:
We should have 2 properties. We currently have themeColor, and your PR should introduce the gitLogArguments.

Reply:
Yeah but interface.js reads that from theme.json and even if we store it within config json, the interface.js still reads it from theme.json. Shall I add theme color to config as well? If so, will updating interface.js and removing theme.json be enough?

Reply:
https://github.com/jwu910/check-it-out/issues/102

I was hoping that this issue would remove the use of theme.json since we're storing only the color value there right now. This ticket should abstract that out and create the config as the alternative to using that json. 

Idealy a user should not need to edit source files to configure the application and your implemetation of the configstore would be a good direction to go.

Reply:
Allrighty, did include themeColor back in.
* Updated interface.js to read from config file
* Removed theme.json
* Removed theme.json from package.json
 I wanted to ask if I need to change something else?

FYI, on Windows with both bash and cmder (conemu) I cannot see theme colors of blessed. If not opened, I will create an issue for that

Reply:
Also updated the readme.md. Above you mentioned a template. Did you mean the config json contents as a template?

## Settings
Configurable settings can be found at `~/.config/configstore/check-it-out.json`

Options include:
* gitLogArguments
	* Type: String
	* Default: `--color=always`
	* Git Log argument to format output
* themeColor
	* Type: String
	* Default: `#FFA66D`
	* A hex color code to style Check It Out

Reply:
Cool, lets get the code up so we can take a look.

Also if you can clean up the older commits, i'd appreciate that too. :+1: